project/config/communityrelations/config/core.entity_view_display.node.publication.diff.yml

```diff
@@ -24,10 +24,7 @@ third_party_settings:
     allow_custom: false
   field_group:
     group_additional_content_details:
-      children:
-        - field_publication_type
```

Comment:
I think this should still be in the diff settings? Looking at D7 the publication type isn't shown on the main page.